.github/workflows/docker_builds.yml

```diff
@@ -19,28 +19,26 @@ jobs:
       if: contains(github.ref, 'refs/tags/')
       id: get_version
       run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
-    - uses: actions/checkout@2.14
+    - uses: actions/checkout@v2
     - name: Publish Releases to Docker
       # only on releases
-      uses: elgohr/Publish-Docker-Github-Action@master
+      uses: elgohr/Publish-Docker-Github-Action@2.14
       if: contains(github.ref, 'refs/tags/') && !contains(${{ steps.get_version.outputs.VERSION }}, 'rc') %% !contains(${{ steps.get_version.outputs.VERSION }}, 'dev')
       with:
         name: pytorchlightning/pytorch_lightning
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
         dockerfile: docker/Dockerfile
         buildargs: PYTHON_VERSION=${{ matrix.python_version }},PYTORCH_VERSION=${{ matrix.pytorch_version }},LIGHTNING_VERSION=${{ steps.get_version.outputs.VERSION }}
-        tags: "${{ steps.get_version.outputs.VERSION }}_py${{ matrix.python_version }}_torch${{ matrix.pytorch_version }},stable_py${{ matrix.python_version }}_torch${{ matrix.pytorch_version }}"
+        tags: "${{ steps.get_version.outputs.VERSION }}-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }},stable-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }}"
     - name: Publish Master
       # publish master
-      uses: elgohr/Publish-Docker-Github-Action@master
+      uses: elgohr/Publish-Docker-Github-Action@2.14
       if: github.event_name == 'push'
       with:
         name: pytorchlightning/pytorch_lightning
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
         dockerfile: docker/Dockerfile
         buildargs: PYTHON_VERSION=${{ matrix.python_version }},PYTORCH_VERSION=${{ matrix.pytorch_version }},LIGHTNING_VERSION=${{ steps.get_version.outputs.VERSION }}
-        tags: "latest_py${{ matrix.python_version }}_torch${{ matrix.pytorch_version }}"
-  
-
+        tags: "latest-py${{ matrix.python_version }}-torch${{ matrix.pytorch_version }}"
```

Comment:
what is the latest? like latest from commit from master?

Reply:
Yes the latest (unstable, not yet released) version (from master in our case)

Reply:
Then I would rather call it nightly, at least my association with latest is stable...

Reply:
or simply unstable? Because nightly for me refers to one build every night (which might not be given here)

Reply:
I would stay with nightly... the nigh is for active repos to reduce nb builds just once overnight in our case we do not have so many commits to master every day so I guess it is the same...
I think if we name it unstable no one will use it :D
but yes, to be consistent we can make a crone for Docker build, right?
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule